java101/pom.xml

```diff
@@ -38,6 +38,11 @@
 			<artifactId>spring-boot-starter-freemarker</artifactId>
 		</dependency>
 		<dependency>
+	        <groupId>org.springframework</groupId>
```

Comment:
Mam problem z tym, bo jak cokolwiek zmieniam w pom.xml (np. wersję spring boota) to wywala mi się klasa Applications..